OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -755,6 +763,17 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
             is_callchain_sample <=
         1);
 
+  absl::flat_hash_map<orbit_grpc_protos::TracepointInfo, bool, internal::HashTracepointInfo,
+                      internal::EqualTracepointInfo>
+      are_instrumented_tracepoints;
```

Comment:
This name is weird. Why "are" in front?

Reply:
It was thought to be more like a question. Such as "are these events the instrumented tracepoints that were selected". But yes, I see how this seems weird and would need to be changed.